Tests/test_fslViewer.py

```diff
@@ -323,10 +325,48 @@ def tearDown(self):
 		self.postStatsFile.close()
 		
 if __name__ == "__main__":
+
 	scriptPath = os.path.dirname(os.path.abspath(__file__)) #Get path of script
-	dataPath = os.path.join(scriptPath, "data")
-	globData = os.path.join(dataPath,"*.ttl")
-	data = glob.glob(globData)
+	dataDir = os.path.join(scriptPath, "data")
+	dataNames = ["fsl_con_f", "fsl_thr_clustfwep05","ex_spm_thr_voxelunct4","ex_spm_thr_clustunck10","ex_spm_thr_voxelfdrp05"]
+	local = True
+	
+	for dataName in dataNames: #Check if data is on local machine
+	
+		if os.path.isfile(os.path.join(dataDir, dataName + ".nidm.ttl")) == False: #Data not found on local machine
+		
+			local = False
+			break
+			
+	if local == False:
+	
+		req = urllib.request.Request("http://neurovault.org/api/collections/1692/nidm_results") #Request from neurovault api
+		resp = urllib.request.urlopen(req)
+		readResp = resp.read()
+		data = json.loads(readResp.decode('utf-8'))
+		
+		for nidmResult in data["results"]:
+		
+			
+			turtUrl = nidmResult["ttl_file"] #Url of turtle file
+			dataName = nidmResult["name"] #Name of data (e.g. fsl_con_f.nidm)
+			
+			if dataName in [d + ".nidm" for d in dataNames]: #Check if data is required for tests
+			
+				turtFile = urllib.request.urlopen(turtUrl)
+				dataPath = os.path.join(dataDir, dataName + ".ttl") 
+				
+				if os.path.isfile(dataPath) == False:
+				
+					dataFile = open(dataPath, "w") 
```

Comment:
Can you add an if condition here to create the `dataDir` folder if it does not exist? Otherwise the following error is raised when there is no `data` folder:
```
$ python Tests/test_fslViewer.py 
Traceback (most recent call last):
  File "Tests/test_fslViewer.py", line 361, in <module>
    dataFile = open(dataPath, "w") 
FileNotFoundError: [Errno 2] No such file or directory: '<PATH>/Tests/data/ex_spm_thr_voxelunct4.nidm.ttl'
```